src/controllers/trainingPlan/index.ts

```diff
@@ -8,8 +8,10 @@ import {
 import { handlePrismaError } from '../../lib/handle-prisma-error';
 
 export default class TrainingPlanController {
-  static async getAll(_: Request, response: Response) {
-    const trainingPlans = await TrainingPlanService.getAll();
+  static async getAll(request: Request, response: Response) {
+    const query = request.query as Record<string, any>;
+    const trainingPlans = await TrainingPlanService.getAll(query);
+
```

Comment:
filter the query. This way i can inject anything I want from the URL and it will filter.

Reply:
filter for what you want. in this case, just creator

```suggestion
  static async getAll(request: Request, response: Response) {
    const query = request.query as Record<string, any>;
    
    const filter = {
      creator_id: query.creator
    }
    
    const trainingPlans = await TrainingPlanService.getAll(filter);

```